governance/governance.go

```diff
@@ -780,23 +773,8 @@ func (w *GovernanceWorker) RecordReply(proposal *citizenscientist.Proposal, prot
 				glog.Errorf("Error: %v", err)
 			}
 			envAdds[config.ENVVAR_PREFIX+"AGREEMENTID"] = proposal.AgreementId
-			envAdds[config.COMPAT_ENVVAR_PREFIX+"AGREEMENTID"] = proposal.AgreementId
-			envAdds[config.ENVVAR_PREFIX+"CONTRACT"] = w.Config.Edge.DVPrefix + proposal.AgreementId
-			envAdds[config.COMPAT_ENVVAR_PREFIX+"CONTRACT"] = w.Config.Edge.DVPrefix + proposal.AgreementId
-			// Temporary hack
-			if workload.WorkloadPassword == "" {
-				envAdds[config.ENVVAR_PREFIX+"CONFIGURE_NONCE"] = proposal.AgreementId
-				envAdds[config.COMPAT_ENVVAR_PREFIX+"CONFIGURE_NONCE"] = proposal.AgreementId
-			} else {
-				envAdds[config.ENVVAR_PREFIX+"CONFIGURE_NONCE"] = workload.WorkloadPassword
-				envAdds[config.COMPAT_ENVVAR_PREFIX+"CONFIGURE_NONCE"] = workload.WorkloadPassword
-			}
 			envAdds[config.ENVVAR_PREFIX+"HASH"] = workload.WorkloadPassword
-			// For workload compatibility, the DEVICE_ID env var is passed with and without the prefix. We would like to drop
-			// the env var without prefix once all the workloads have ben updated.
-			envAdds["DEVICE_ID"] = w.deviceId
```

Comment:
I am not sure if DEVICE_ID should be deleted or not. Need to confirm from @bmpotter .